lib/client.js

```diff
@@ -12,16 +12,26 @@ module.exports = async function connectClient (instance, log, options) {
   }
 
   const client = await CDP({port: instance.port});
-  const { DOM, DOMStorage, Console, Network, Page, Runtime } = client;
+  const { DOM, DOMStorage, Console, Network, Page, Runtime, Security } = client;
   const mochaOptions = `window.mochaOptions = ${JSON.stringify(options.mocha)}`;
 
+  Security.certificateError(({eventId}) => {
+    Security.handleCertificateError({
+      eventId,
+      action: 'continue'
```

Comment:
this should really be an option, which can be set to `continue` by default. perhaps `sslErrorAction`